NEXM84/Utility/PasswordManager.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NEXM84.Utility
+{
+    internal sealed class PasswordManager : Infrastructure.IPasswordManager
+    {
+        private static readonly byte[] Key = new byte[]
+        {
+        0x4E, 0xA1, 0x6F, 0x7B, 0x6A, 0x54, 0x2E, 0x0F, 0x43, 0x97, 0x26, 0x3B, 0xC5, 0x92, 0xAE, 0xD8
+        };
+
+        private static readonly byte[] IV = new byte[]
+        {
+        0x6D, 0x5F, 0x92, 0xE9, 0xC4, 0x12, 0x75, 0xF0, 0xA3, 0x8B, 0x07, 0x4E, 0x9F, 0x63, 0xD7, 0x1A
+        };
+
+        public string AESEncryptPassword(string password)
+        {
+            using (Aes aesAlg = Aes.Create())
+            {
+                aesAlg.Key = Key;
+                aesAlg.IV = IV;
```

Comment:
Konstans IV és KEY beégetve a szoftverbe?? Ez így ebben a formában plain text extra lépésekkel. Ez nagyon, de nagyon nem biztonságos így ebben a megoldásban. https://csharptutorial.hu/docs/hello-vilag-hello-kriptografia/

Reply:
Igen éreztem, hogy az úgy nagyon nem lesz jó. Nem akartam kicsúszni az időből, így nem néztem kellően utána a használatának. (Idő menedzsment terén van hova fejlődnöm...)
Nem tartom pofátlanságnak. Eddig még nem foglalkoztam ilyesmivel, de most kicsit elkezdett érdekelni és meg fogom nézni.
Köszönöm szépen:)